docs/source/portfolio_optimisation/theory_implied_correlation.rst

```diff
@@ -0,0 +1,164 @@
+.. _portfolio_optimisation-theory_implied_correlation:
+
+
+================================
+Theory-Implied Correlation (TIC)
+================================
+This TIC class includes an algorithm to calculate the Theory-Implied Correlation and a method to calculate the correlation matrix distance proposed by Herdin and Bonek. This distance may be used to measure to which extent the TIC matrix has blended theory-implied views (tree structure of the elements) with empirical evidence (correlation matrix).
+
+About the Algorithm
+###################
+
+The TIC algorithm is aiming to estimate a forward-looking correlation matrix based on economic theory. The method is using a theoretical classification of assets (hierarchical structure) and fits the empirical correlation matrix to the theoretical structure.
+
+According to Lopez de Prado, author of the algorithm: "A problem of empirical correlation matrices is that they are purely observation driven, and do not impose a structural view of the investment universe, supported by economic theory."
```

Comment:
Add a link to this paper. You can do so by following this syntax ``` `text <link>`_ ```

Reply:
Specified the paper used for quote